developer_manual/app/changelog.rst

```diff
@@ -0,0 +1,39 @@
+=========
+Changelog
+=========
+
+.. sectionauthor:: Bernhard Posselt <dev@bernhard-posselt.com>
+
+The following changes went into ownCloud 8:
+
+
+Breaking changes
+================
+
+* Setting session variables inside a controller requires the **@UseSession** annotation, otherwise the session will be closed
+
+Deprecations
+============
+
+.. note:: Deprecations on interfaces also affect the implementing classes!
```

Comment:
Can try tomorrow if you didn't do it by then:
https://gist.github.com/nickvergessen/8bb0d870942900bf591d